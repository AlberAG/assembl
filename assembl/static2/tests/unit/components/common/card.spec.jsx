import React from 'react';
import renderer from 'react-test-renderer';

import Card from '../../../../js/app/components/common/card';

describe('Card component', () => {
  it('should match Card snapshot', () => {
    const rendered = renderer
      .create(
        <Card className="classFoo" imgUrl="http://foo.bar/imgUrl">
          <div>Card content</div>
        </Card>
      )
      .toJSON();
    expect(rendered).toMatchSnapshot();
  });
});