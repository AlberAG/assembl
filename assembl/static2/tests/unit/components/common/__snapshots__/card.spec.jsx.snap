// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component should match Card snapshot 1`] = `
<div
  className="illustration-box classFoo"
>
  <div
    className="image-box"
    style={
      Object {
        "backgroundImage": "url(http://foo.bar/imgUrl)",
      }
    }
  />
  <div>
    Card content
  </div>
  <div
    className="color-box"
  >
     
  </div>
  <div
    className="box-hyphen"
  >
     
  </div>
  <div
    className="box-hyphen rotate-hyphen"
  >
     
  </div>
</div>
`;
